src/components/Button/Button.stories.tsx

```diff
@@ -99,15 +97,13 @@ export const Loading = Template.bind({});
 Loading.args = {
   loading: true,
   disabled: true,
-  text: 'Loading...',
+  text: 'Loading Button',
```

Comment:
I changed the `text` in the loading stories because I thought it kinda looked like the `Button` logic was replacing the `text` with "Loading..." because `loading` was `true`.